crawler.py

```diff
@@ -78,6 +82,8 @@ def do_something(driver):
     assert len(body) == 1
     body = body[0]
 
+    body.send_keys(Keys.CONTROL, 0)
```

Comment:
Why this change?

Reply:
`Element <a class="js-gps-track -link" href="/help"> is not clickable at point (135,655.7666625976562) because another element <div id="js-gdpr-consent-banner" class="p16 bg-black-700 fc-white ps-fixed b0 l0 r0 z-banner"> obscures it` - I was getting such kind of exceptions and Stackoverflow suggested that this can be an issue of incorrect (not 100%) zooming.

Reply:
Was the issue fixed with this?

Reply:
Not fully but it decreased the number of occurrences.

Reply:
OK!